### DISCUSSION TITLE
CAD drawings for the F/T sensors

### DISCUSSION THREAD

**Discussion by iron76** [robotology/community#55] [Q&A] [answered]
I am trying to find the CAD drawings for the F/T sensors described here (http://wiki.icub.org/wiki/FT_sensor). Can anyone give me a pointer to repository were these drawings can be downloaded?
Thanks in advance.

**Comment by traversaro** [comment]
I think it should be this one: https://svn.robotology.eu/repos/iCubHardware-pub/trunk/mechanics/public/icub1_icub2/ftsensor45a.asm.1 .

**Comment by iron76** [comment]
What if I only need to share with someone else only the dimensions and the screw positions for designing a socket for the FT sensor? Is there an simpler format to share?

**Comment by traversaro** [comment]
Then I think I am unable to reply, and we need some help from some mechanics expert (@fiorisi ? ).

**Comment by fiorisi** [comment, marked as answer]
You can find the dxf files for the ftsensor45 in this folder:
https://svn.robotology.eu/repos/iCubHardware/trunk/mechanics/public/icub1_icub2/prodoc

where:
- ftsensor45a.dxf                  --> assembly 
- ftsensor45.dxf                    --> sensor cell
- bottomcover45_r_A1.dxf   --> bottom cover
- topcover45.dxf                   --> top cover

**Comment by iron76** [comment]
@fiorisi, can we also have the details of the screws you are using to mount the FT sensor. Is the diameter M2.5? Thanks in advance.

**Comment by fiorisi** [comment]
Both the top and bottom cover have M3 holes.

**Comment by fiorisi** [comment]
![ftsens](https://cloud.githubusercontent.com/assets/8970230/10390085/913b8bd6-6e76-11e5-8f91-e787d57dfee8.png)

**Comment by pattacini** [comment]
@iron76, is @fiorisi's answer satisfactory?

**Comment by iron76** [comment]
Yes, I will close the issue.
